apps/bynder/README.md

```diff
@@ -0,0 +1,23 @@
+## Bynder Assets
```

Comment:
I'd just create a simple README per app pointing to the docs. This file mentions UI Extensions and is outdated